src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -218,6 +219,7 @@ const AppWrapper = (props: { children: ReactNode }) => {
       // Agent.agent.credentials.syncACDCs(),
```

Comment:
Maybe not related to this PR, but just noticed this comment is still here. Not sure if you are tracking it separately @iFergal.

Reply:
yep I am, thanks